index.js

```diff
@@ -14,6 +14,10 @@ function bad_style() {
     return 'Yes';
 }
 
+function bad_style2() {
```

Comment:
Identifier 'bad_style2' is not in camel case.<br>'bad_style2' is defined but never used.